libfuse/file.go

```diff
@@ -11,19 +11,41 @@ import (
 	"golang.org/x/net/context"
 )
 
+type eiCache struct {
+	ei  libkbfs.EntryInfo
+	rid string
+}
+
 // File represents KBFS files.
 type File struct {
 	folder *Folder
 	node   libkbfs.Node
+
+	// for the Attr call after Create
+	eiCache *eiCache
```

Comment:
I mean if this happens:

1. Bazil sends Create call, and it completes.
2. Another thread/process writes to the file somehow.
3. Bazil sends the corresponding Attr call.

I'm not sure if it's possible for (2) to sneak in between (1) and (3) -- maybe there's no way to get a file handle before the overall Create syscall returns.  But it might be possible, and maybe worth guarding against.